lib/secure_headers/headers/content_security_policy.rb

```diff
@@ -101,12 +101,17 @@ def build_media_type_list_directive(directive)
     # directive_name - a symbol representing the various ALL_DIRECTIVES
     #
     # Returns a string representing a directive.
+    NEWLINE_OR_SEMI_COLON = /(\n|;)/
     def build_source_list_directive(directive)
       source_list = @config.directive_value(directive)
       if source_list != OPT_OUT && source_list && source_list.any?
-        minified_source_list = minify_source_list(directive, source_list).join(" ")
+        minified_source_list = if @config[:disable_minification]
```

Comment:
If `disable_minification` is true, then it doesn't look like `populate_nonces` gets called. Is that an issue?

Reply:
> If `disable_minification` is true, then it doesn't look like `populate_nonces` gets called. Is that an issue?

Definitely. This blunt approach needs a little more care/definition. I'll add that to the list.